### PR TITLE
Update for 3.1.0 fixes for isAtEnd.

### PR DESCRIPTION
**Pull request by mbeckerle**
Removed use of isAtEnd. Not needed.

### CODE REVIEW

File: hexWords/build.sbt [hunk 1]
```diff
@@ -7,8 +7,8 @@ version := "0.0.1"
 scalaVersion := "2.12.11"
 
 libraryDependencies ++= Seq(
-  "org.apache.daffodil" %% "daffodil-tdml-processor" % "3.0.0",
-  "org.apache.daffodil" %% "daffodil-japi" % "3.0.0",
+  "org.apache.daffodil" %% "daffodil-tdml-processor" % "3.1.0-SNAPSHOT",
+  "org.apache.daffodil" %% "daffodil-japi" % "3.1.0-SNAPSHOT",
```
Comment (stevedlawrence) [inline hexWords/build.sbt:11]: Thoughts on only depending on a snapshot release on a dev branch? I'm thinking we want to make sure that these examples always work with ``sbt test`` without having to publishLocal a custom build of daffodil.

**Review by stevedlawrence — APPROVED** [review]
+1, might want to merge into a dev branch until 3.1.0 is released?

File: hexWords/build.sbt (hunk 1 above)
Comment (mbeckerle) [reply hexWords/build.sbt:11]: Agreed. I don't think I can switch branches during a PR. Will close this one and start a new one on a dev branch.

**Comment by mbeckerle** [comment]
Redoing with a fix, and on a dev-branch instead of master.

**Pull request closed without merging**